lib/ansible/plugins/lookup/ini.py

```diff
@@ -43,6 +43,12 @@
           duplicate key errors if keys only differ in case.
         default: False
         version_added: '2.12'
+      allow_none:
+        description:
+        - Read ini file which contains key without value and without '=' symbol.
+        type: bool
+        default: False
+        version_added: '2.12'
```

Comment:
Add an alias to make this match the `community.general.ini` module, which calls this `allow_no_value`.